worker/workercmd/guardian.go

```diff
@@ -31,13 +31,7 @@ func (cmd *WorkerCommand) guardianRunner(logger lager.Logger) (ifrit.Runner, err
 
 	members := grouper.Members{}
 
-	gdnFlags := []string{}
-
-	if cmd.Garden.Config.Path() != "" {
```

Comment:
We might want to keep this even though it could be passed directly to guardian via the detectGuardianFlags as it helps operators by showing up in `concourse worker -h` and it has been there [historically](https://github.com/concourse/concourse/commit/a5323c4de6d4e4d6b6d1fb783772e7a1b31cdf25)

Reply:
I agree, having it be explicitly stated as a flag makes it more trustworthy and easier to discover than having it be implicit.

Even the [docs](https://concourse-ci.org/concourse-worker.html#configuring-gdn-server) say that it's more for backwards compatibility than as the preferred way of configuring garden

Reply:
Great catch! Changed it back to what it was. Also renamed `gdnConfigFlags` to explicitly call out that it's being used for the `--config` option.